guides/release-process.md

```diff
@@ -164,7 +172,9 @@ In the following instructions, "X.Y.Z" is used to denote the [next version of Cy
 
```

Comment:
For step 19. I had to do a `--force` to get dependencies installed and also needed to pass the `releaseData.json` as an absolute path. It might be good to capture that here if you had a similar experience?

Reply:
@AtofStryker were you using the project's expected node version?

Reply:
I think it failed to activate the `lts` from `avn`, so my guess would be no. I couldn't download the LTS from node due to the tarball missing?  I wonder if its good to prompt users to run `nvm install --lts` since the version could be changing frequently.

Reply:
I don't think its a node version issue though, but a conflicting peer dep?
![Screenshot 2023-03-16 at 11 20 27 AM](https://user-images.githubusercontent.com/3980464/225663990-4fe0007f-3e90-4c86-a8db-58a6632710f6.png)